src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return 80;
+  } else if (days >= 3 && days < 6) {
+    return 100;
```

Comment:
This condition is incorrect as it does not consider the discount for renting 3 or more days. You need to calculate the total cost and then subtract the discount if applicable.